.github/workflows/performance_windows.yml

```diff
@@ -3,6 +3,8 @@ name: PerformanceWindows
 on:
   push:
     branches: [ development ]
+  pull_request:
```

Comment:
this is done just for performance testing pre-merge...; this will be reverted for the actual merge